client/tests/rococo.test.ts

```diff
@@ -8,5 +8,5 @@ const chains = [
   { name: "Bridgehub", id: 1013 },
 ];
 
-const test = new FaucetTests("/", "PUBLIC_FAUCET_ROCOCO_URL", "Rococo Faucet", chains);
-test.runTests();
+const tests = new FaucetTests({ faucetName: "Rococo Faucet", chains, url: "/", expectTransactionLink: true });
```

Comment:
imo would be cool to be explicit there, and instead of root "/" have here "/rococo"
then in the root of app show 3 (or how many) links/cards to supported networks.. so whoever lands there can click needed network

Reply:
yes, but they still can point directly to a needed faucet, so there will be same amount of clicks, you just provide /rococo in appropriate docs, but there will be better navigation and clear distinction 

who is "they" btw? )

Reply:
I agree with Mak, it also bugs me a bit.  
Wouldn't do it here though, as it has to be updated in several places.

Reply:
I'll write a ticket and we can go to design with a requirement for it

Reply:
Done: https://github.com/paritytech/polkadot-testnet-faucet/issues/329